api/async.rb

```diff
@@ -112,18 +112,24 @@ def validate
   # polled against. This class is meant to poll the resource itself until it can be
   # determined that the resource is in a ready state.
 
-  # NOTE - this class is not fully generated and requires the polling implementation to
-  # be written by hand per product
+  # NOTE - this class is not fully generated and may require the polling
+  # implementation to be written by hand per product. If match_response_state
```

Comment:
Where is `match_response_state` defined?

Reply:
old comment, I'll remove